docs/guides/nextauth-migration.mdx

```diff
@@ -0,0 +1,362 @@
+---
+title: Migrating to Clerk from NextAuth.js
+description: Migrating from NextAuth.js
+---
+
+<TutorialHero
+  framework="next"
+  beforeYouStart={[
+    {
+      title: "Set up a Clerk application",
+      link: "https://clerk.com/docs/quickstarts/setup-clerk",
+    }
+  ]}
+>
+
+- Install `@clerk/nextjs`
+- Set environment keys
+- Wrap your Next.js app in `<ClerkProvider />`
+- Setup Sign-up and Sign-in UI
+- Protect your app
+- Read user and session data
+- User IDs as Foreign Keys
+- Create a production instance
+- Migrate users to a production instance
+- Further support
+
+</TutorialHero>
+
+This guide shows how to migrate an application using NextAuth.js to use Clerk for authentication.
+
+<Steps>
+
+### Install `@clerk/nextjs`
+
+Clerk's Next.js SDK gives you access to prebuilt [components](https://clerk.com/docs/components/overview), [hooks](https://clerk.com/docs/references/nextjs/overview#client-side-helpers), and [helpers](https://clerk.com/docs/references/nextjs/overview) for Next.js Server Components, Route Handlers and Middleware. Install it by running the following command in your terminal:
+
+<CodeBlockTabs type="installer" options={["npm", "yarn", "pnpm"]}>
+  ```bash filename="terminal"
+  npm install @clerk/nextjs
+  ```
+
+```bash filename="terminal"
+yarn add @clerk/nextjs
+```
+
+```bash filename="terminal"
+pnpm add @clerk/nextjs
+```
+
+</CodeBlockTabs>
+
+### Set environment keys
+
+Store the keys in the `.env.local` file of your Next.js application.
+
+<InjectKeys>
+
+```sh filename=".env.local"
+NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY={{pub_key}}
+CLERK_SECRET_KEY={{secret}}
+```
+
+</InjectKeys>
+
+### Wrap your Next.js app in `<ClerkProvider />`
+
+#### Remove your NextAuth.js `<SessionProvider />`
+
+Replace `<SessionProvider session={session}>` with `<ClerkProvider>` if you are using App Router or `<ClerkProvider {...pageProps}>` if you are using Pages Router.
+
+```tsx filename="_app.tsx"
+// With NextAuth.js
+
+import { SessionProvider } from "next-auth/react"
+
+export default function App({
+  Component,
+  pageProps: { session, ...pageProps },
+}) {
+  return (
+    <SessionProvider session={session}>
+      <Component {...pageProps} />
+    </SessionProvider>
+  )
+}
+```
+
+#### Add `<ClerkProvider />` to your root `layout.tsx` file
```

Comment:
I'd replace the `<h4>` with a numbered list instead since these are steps someone has to follow.